tests/python_build/fixtures/copy.glsl

```diff
@@ -0,0 +1,59 @@
+/* clang-format off */
```

Comment:
Not sure if it matters, but this specific example uses syntax specific to the GLES3 shaders, but the changed file is the builder for the Vulkan GLSL shaders. 

I'm sure it doesn't matter right now as you are just validating the fact that it copies the shader without making changes, But if there is a chance the test is going to get more complex/specific, we may want to use a different shader as an example.

Reply:
thanks for comment. Which file would you recommend to use?

Reply:
Any of the ones under [servers/rendering/renderer_rd/shaders](https://github.com/godotengine/godot/tree/master/servers/rendering/renderer_rd/shaders) would be fine. If length isn't an issue [canvas.glsl](https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/shaders/canvas.glsl) would be a good one to use as it uses most features (includes, inserting shader code, version defines)

Reply:
ok, I force-pushed (to keep the commits tidy) `canvas.glsl`

Reply:
I think I'd prefer a smaller, carefully crafted `.glsl` file that uses all relevant features we support in codegen, but without too much "real code" that adds quite a few unnecessary lines of code to the codebase (and very long lines of bytes in the generated file).

If we could keep it to around 100 LOCs or something that would be nice. Having pseudocode should also help prevent getting hits in the test case when grepping for things present in `canvas.glsl` to find occurrences of a potential bug.

Reply:
> I think I'd prefer a smaller, carefully crafted .glsl

that makes sense, alas at this point I have zero insight into what might be considered "relevant features".

What I touched within this PR is affecting generator of compute/vertex/fragment shader, so the best I can do is to take a small existing compute shader (looking at [roughness_limiter.glsl](https://github.com/godotengine/godot/blob/e7197cb6f985eaa0a898ed1b0991d88db2f1312b/servers/rendering/renderer_rd/shaders/roughness_limiter.glsl) with ~50 lines without comments) and vertex/fragment shader (looking at [specular_merge.glsl](https://github.com/godotengine/godot/blob/e7197cb6f985eaa0a898ed1b0991d88db2f1312b/servers/rendering/renderer_rd/shaders/specular_merge.glsl) with ~50 lines) and use these under a different name until something carefully crafted will be supplied instead.

Reply:
Yeah I'm more thinking that such a test shader could be provided by @clayjohn or other folks in @godotengine/rendering - if they agree with my proposal to use a pseudocode one. Otherwise I'm happy to defer to their judgement if they think it's best to use a real world shader.

Reply:
@clayjohn unless it's somehow obfuscated, I dont see anything being done with `#CODE` macro in the Python builder files. The only place I can find it is in `shader_gles3.cpp` which is out of scope of what I am trying to do here:
https://github.com/godotengine/godot/blob/afdae67cc381bb340da2e864279da6b836804b7f/drivers/gles3/shader_gles3.cpp#L66